spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do 
+  before(:each) do
+    @factory = FacilityFactory.new
+    @service = DmvDataService.new
+
+    @co_dmv_data = @service.co_dmv_office_locations
+    @ny_dmv_data = @service.ny_dmv_office_locations
+  end
+
+  describe '#create_facilities' do
```

Comment:
These are good, robust tests.  I really like that you are verifying not only that a Facility exists, but also that it is the exact facility that you expect.